packages/plugin-e2e/src/fixtures/commands/createAppConfigPage.ts

```diff
@@ -0,0 +1,23 @@
+import { TestFixture } from '@playwright/test';
+import { PluginFixture, PluginOptions } from '../../api';
+import { PluginConfigPageArgs } from '../../types';
+import { PlaywrightCombinedArgs } from '../types';
+import { AppConfigPage } from '../../models/pages/AppConfigPage';
+
+type CreateAPPConfigPageFixture = TestFixture<
+  (args: PluginConfigPageArgs, options?: { goto?: boolean }) => Promise<AppConfigPage>,
+  PluginFixture & PluginOptions & PlaywrightCombinedArgs
+>;
+
+export const createAppConfigPage: CreateAPPConfigPageFixture = async (
+  { page, selectors, grafanaVersion, request },
+  use
+) => {
+  await use(async (args, options = { goto: true }) => {
+    const appConfigPage = new AppConfigPage({ page, selectors, grafanaVersion, request }, args);
+    if (options.goto) {
```

Comment:
I like those suggestions and I agree with the option to create the AppConfigPage directly instead of passing options to the create function. I will apply those changes.